integration-tests/tasks/rhtap-install.yaml

```diff
@@ -57,7 +57,11 @@ spec:
         set -o nounset
         set -o pipefail
 
-        GIT_REPO="${GIT_REPO:-$(echo "$JOB_SPEC" | jq -r '.git.git_repo')}"
+        GIT_REPO="${GIT_REPO:-$(echo "$JOB_SPEC" | jq -r '.git.repo // empty')}"
+        if [ -z "$GIT_REPO" ]; then
```

Comment:
I see I missed this change in my PR... So - maybe merge my PR and change this PR to contain this single fix and merge it then?

Reply:
sure, I will close this PR